### PR TITLE
added the children classes of article

### PR DESCRIPTION
**Pull request by ace-spadez**

### CODE REVIEW

**Comment by ace-spadez** [comment]
i change the ccontent field   type   from text to charfield(limit=256)

File: article/models.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
     author = models.ForeignKey("auth.User",on_delete = models.CASCADE,verbose_name = "Author")
     title = models.CharField(max_length = 50,verbose_name = "Title")
-    content = models.TextField()
+    # content = RichTextField()
+    content =models.CharField(max_length=256,verbose_name="Content")
+
     created_date = models.DateTimeField(auto_now_add=True,verbose_name="Created Date")
     article_image = models.FileField(blank = True,null = True,verbose_name="Image")
+    treat =  models.ForeignKey(Treat,on_delete=models.PROTECT,verbose_name="Treat",  blank=True,null=True)
+    valid  = models.BooleanField(default=True,verbose_name="Validity")
```
Comment (orientor) [inline article/models.py]: Why did you add treat and valid to article?

File: article/models.py [hunk 2]
```diff
@@ -1,12 +1,17 @@
 from django.db import models
 # Create your models here.
-
+class Treat(models.Model):
+    title =  models.CharField(max_length=20,verbose_name="Title")
 class Article(models.Model):
     author = models.ForeignKey("auth.User",on_delete = models.CASCADE,verbose_name = "Author")
     title = models.CharField(max_length = 50,verbose_name = "Title")
-    content = models.TextField()
```
Comment (orientor) [inline article/models.py]: I think TextField() will be better than charField as content is usually more than 256 characters.
